### PR TITLE
Add the initializer to e2e cluster wide test

### PR DESCRIPTION
**Pull request by andraxylia**
Add the initializer to e2e cluster wide test

### CODE REVIEW

File: tests/e2e/framework/kubernetes.go [hunk 1]
```diff
@@ -278,6 +278,9 @@ func (k *KubeInfo) deployIstio() error {
 			glog.Errorf("Istio initializer %s deployment failed", testInitializerYAML)
 			return err
 		}
+
+		// alow time to the initializer to start
+		time.Sleep(60 * time.Second)
```
Comment (ayj) [inline tests/e2e/framework/kubernetes.go]: optional: `s/60 * time.Second/time.Minute`

**Comment by ldemailly** [comment]
doesn't seem to pass the new- test

**Comment by ldemailly** [comment]
@ayj / @andraxylia  this didn't pass the new- test, see https://k8s-gubernator.appspot.com/pr/istio/istio/933

**Comment by ldemailly** [comment]
we shouldn't let the bot merge a test whose goal is to pass a new test when said new test doesn't pass

**Comment by ayj** [comment]
The bookinfo app is being installed in the `istio-system` namespace which is exempt from injection.

```
$ kubectl -n istio-system get deployment 
NAME                DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
details-v1          1         1         1            1           2m
istio-ca            1         1         1            1           4m
istio-egress        1         1         1            1           4m
istio-ingress       1         1         1            1           5m
istio-initializer   1         1         1            1           3m
istio-mixer         1         1         1            1           6m
istio-pilot         1         1         1            1           5m
mongodb-v1          1         1         1            1           2m
productpage-v1      1         1         1            1           2m
prometheus          1         1         1            1           2m
ratings-v1          1         1         1            1           2m
ratings-v2          1         1         1            1           2m
reviews-v1          1         1         1            1           2m
reviews-v2          1         1         1            1           2m
reviews-v3          1         1         1            1           2m
zipkin              1         1         1            1           2m
```

**Comment by sebastienvas** [comment]
That s more a human mistake than a bot issue.

**Comment by sebastienvas** [comment]
Should we revert ?

**Comment by andraxylia** [comment]
Let me submit a PR to revert until I can move the bookinfo to another namespace.

**Comment by ldemailly** [comment]
revert was https://github.com/istio/istio/pull/953

we need to make the test visible (and maybe compulsory too if it's not flaky)

**Comment by sebastienvas** [comment]
@yutongz Could you make the job visible ?

**Comment by andraxylia** [comment]
When you do this, can you please also rename the script to new-e2e-cluster-wide-auth or similar? We can remove rbac from the name, everything has rbac nowadays.

**Comment by yutongz** [comment]
https://github.com/istio/test-infra/pull/523
https://github.com/istio/istio/pull/966

**Pull request merged**